src/python/foglamp/rest_service/routes.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+from foglamp.rest_service import api
+
+
+def setup(app):
+    # app.router.add_route('POST', '/foglamp/a-post-req', api.a_post_req, expect_handler = aiohttp.web.Request.json)
+    app.router.add_route('GET', '/foglamp/ping', api.ping)
+
+    # enable cors only in dev mode ?!
+    enable_cors(app)
+    # a live debugger (watcher) for requests
+    # this will neutralize error middleware
+    enable_debugger(app)
+
+
+def enable_cors(app):
+    import aiohttp_cors
+
+    # Configure default CORS settings.
+    cors = aiohttp_cors.setup(app, defaults={
+        "*": aiohttp_cors.ResourceOptions(
+            allow_credentials=True,
+            expose_headers="*",
+            allow_headers="*",
+        )
+    })
+
+    # Configure CORS on all routes.
+    for route in list(app.router.routes()):
+        cors.add(route)
+
+
+def enable_debugger(app):
+    """ provides a debug toolbar for server requests """
+    import aiohttp_debugtoolbar
```

Comment:
This allows us to watch requests being made to the rest (aiohttp) server and profiling.. we need this in dev mode only.
![aiohttp_debugtoolbar](https://user-images.githubusercontent.com/1974003/27706645-e8cf091e-5d2f-11e7-939b-295bf3092ec5.png)
![ping_response](https://user-images.githubusercontent.com/1974003/27706647-eaefee48-5d2f-11e7-832b-aebce54ab0a1.png)